semperfiwebdesign/all-in-one-seo-pack
# Don't output default robots meta tag value

Issue #2768

## Proposed changes

Don't output the robots meta tag when its value is "index,follow".

## Types of changes

- Improves existing code

## Checklist


## Testing instructions

1. Publish a new post or page and make sure the robots meta tag isn't output in the HEAD.
2. Either noindex/nofollow the page (or both) and make sure the robots meta tag does show up.
You'll also want to test the global settings.

Have a look at the code to see how it's filtered.
